shared/src/i18n/en/shared.json

```diff
@@ -1,7 +1,23 @@
 {
-  "title": "Welcome to react using react-i18next and typescript 4.1",
-  "description": {
-    "part1": "This is an example using multiple namespaces.",
-    "part2": "😉"
+  "ProductCopies": {
+    "calldescription": "Generates yield by running an automated {{asset}} covered call strategy.",
```

Comment:
Would it be better for us to camelcase these?

Reply:
Sure we can.  It doesn't matter - style preference.
Maybe we camelcase with first letter capitalized to be consistent for all.

Reply:
Yeap this works, will be consistent with everything if so